tests/framework/cli/test_starters.py

```diff
@@ -490,6 +491,16 @@ def test_fail_if_dir_exists(self, fake_kedro_cli):
         assert result.exit_code != 0
         assert "directory already exists" in result.output
 
+    def test_cookiecutter_exception_if_no_verbose(self, fake_kedro_cli):
+        """Check if original cookiecutter exception present in the output if no verbose
+        flag provided."""
+        KedroCliError.VERBOSE_EXISTS = True
```

Comment:
Does that mean to reset the side-effect from the other test? If so I think we make sure this happen after that test so when we add new test in the future or reordering tests won't fail again.